### PR TITLE
add example setup for a vue.js application

### PR DESCRIPTION
**Pull request by phoet**
since there is no vue.js plugin right now, i tried to come up with a minimal example of how one could use it with vue.js

### CODE REVIEW

**Comment by xadamy** [comment]
@phoet Thank you so much for this contribution! 🎉 

This looks great to me, so I'm very happy to merge this! Just FYI, however, I am working on the Vue.js plugin as we speak, so expect it to land in the coming weeks. Once it's landed, I will update this page with the documentation for the new plugin.

**Comment by phoet** [comment]
@xadamy sure thing! if there is some code to test, let me know, we have a good staging system where i can savely test it in a complicated app.

**Comment by jvanbaarsen** [comment]
@phoet Thanks for your contribution. Can you make sure this page is also linked in the navigation?

File: source/front-end/integrations/vue.html.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Import the code in your App:
+
+```js
+// main.js
+import 'appsignal';
+// in case you would like to use the client in your app
+// import appsignal from 'appsignal';
+```
+
+:tada:
```
Comment (jvanbaarsen) [inline source/front-end/integrations/vue.html.md]:
```suggestion
```

This is not being rendered in our docs, maybe better to remove it.

File: source/front-end/integrations/vue.html.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: "AppSignal for JavaScript - Vue.js"
+---
+
+## Installation
+
+Add the `@appsignal/javascript` package to your `package.json`. Then, run `yarn install`/`npm install`.
```
Comment (jvanbaarsen) [inline source/front-end/integrations/vue.html.md]: In our docs we generally use double quotes, and no trailing `;`.

Comment (phoet) [reply source/front-end/integrations/vue.html.md]: dude, airbnb styleguide!!!111 😸

**Comment by phoet** [comment]
👍

**Comment by xadamy** [comment]
@phoet Just wanted to update you on progress here: you can track my merge request for the Vue integration [here](https://github.com/appsignal/appsignal-javascript/pull/17), which should be released sometime this week.

Also, we'd love to send you some stroopwafels by way of thanks for your contribution! 🎉 Would you be able to get in touch with support@appsignal.com with your shipping address (or just let us know if you'd like us to just send them to your billing address)?

**Comment by phoet** [comment]
@xadamy billing address works, everybody loves stroopwafels! 😍

**Pull request merged**